kitura-basic.js

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const tools = require('./helper');
+
+const gitURL = 'https://github.com/IBM-Swift/generator-swiftserver-projects'
```

Comment:
I thought we needed to end lines with semi-colons to preserve compatibility with older Node.js versions... I might be wrong though.

Reply:
Those lines were copied from what we already had. 
`kitura init` didn't end those lines with a semi-colon either however I think we should as it is regarded as good practice.